honeybee_grasshopper_energy/src/HB ConstructionSet.py

```diff
@@ -0,0 +1,129 @@
+# Honeybee: A Plugin for Environmental Analysis (GPL)
+# This file is part of Honeybee.
+#
+# Copyright (c) 2019, Ladybug Tools.
+# You should have received a copy of the GNU General Public License
+# along with Honeybee; If not, see <http://www.gnu.org/licenses/>.
+# 
+# @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+
+"""
+Create a ConstructionSet object containing all energy constructions needed to
+create an energy model. ConstructionSets can be assigned to Honeybee Rooms to
+specify all default constructions on the Room.
+-
+
+    Args:
```

Comment:
I definitely do not like the second one since it is harder for new users to understand all of the constructions they need to input in order to make a full set, which is critical for people to be able to apply them correctly.
Can you explain why you feel these arrangements are better than the current one? 

As an example, I'll give my reasoning for the current setup:
There are primarily two use-cases where I imagine people will want to establish new construction sets:
1. They want to make a construction set that is in accordance with a certain energy standard, building code, or basis-of-design throughout their office. Typically such energy codes/standards are only concerned with properties of exterior constructions and so, to make a valid construction set that reflects such a standard, users must input constructions for all of the inputs of "HB ConstructionSet" in order to know that all default exterior constructions will be overridden when the new set is applied.
2. They want to edit the interior constructions of a set to better account for thermal mass and interior finishes in either a HVAC sizing calculation or a thermal comfort study.  For this case, users can use the "HB ConstructionSet Interior" to override all of these default interior constructions.

Reply:
I see your logic there but I'm not sure if we can generalize that to everyone. For instance I can tell you for a lot of studies for curtain wall prototype they will only change the glass material. Also changing ground material is not as common. But regardless I think breaking them down into categories is more structured and will provide all that you are trying to provide right now. It is also consistent with the other components like program type. I agree that the first one is the better of the two.

Reply:
I just implemented this one.

Reply:
nice! thanks.